x-pack/plugins/enterprise_search/server/routes/workplace_search/groups.test.ts

```diff
@@ -22,9 +22,6 @@ describe('groups routes', () => {
 
     beforeEach(() => {
       jest.clearAllMocks();
-    });
-
-    it('creates a request handler', () => {
       mockRouter = new MockRouter({
         method: 'get',
         path: '/api/workplace_search/groups',
```

Comment:
For future tests, please define the mockRouter and register your routes in the `beforeEach()` blocks, not in `it()` blocks - this makes tests much more easily extendible if you need to add more than 1 test or if you need to add validation tests.